code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -156,6 +157,15 @@
 	var/dead_on_map = FALSE //if true, kills the mob when it spawns (it is for mapping)
 	var/vehicle_version = null
 
+	/**
+	 * List of fluff characteristics to be found on cellular analysis
+	 */
+	var/list/sample_data = list("Cellular structure typical of medium sized fauna")
+	/**
+	 * Whether this mob has cells to be sampled
+	 */
+	var/has_cells = TRUE
```

Comment:
this var should be better named
what "cells"? it is not really apparent that this is about this bio-sampling stuff

Reply:
what is the point of this var, anyways? 
why not just check if `sample_data` is null?